scripts/get_public_key.py

```diff
@@ -0,0 +1,22 @@
+import binascii
+
+from ledgerblue.comm import getDongle
+from ledgerblue.commException import CommException
+
+
+bipp32_path = ( # example of bip32 path
+      "8000002C"
+    + "80000378"
+    + "80000000"
+    + "00000000"
+    + "00000000"
```

Comment:
Is the final value meant to be `"8000002C80000378800000000000000000000000"` ?

```sh

$ python -c 'v = (
> "8000002C"
>     + "80000378"
>     + "80000000"
>     + "00000000"
>     + "00000000"
> ); print(v)'
8000002C80000378800000000000000000000000
```

Reply:
Yes
I have divided into pieces to see parts of bip path clearly